### ISSUE TITLE
[BUG]: No RetroAchievements notifications or interface but still can use feature on Mac

### ISSUE THREAD

**Issue by trvrplk** [PCSX2/pcsx2#7149]
### Describe the Bug

I have no problem earning achievements, but there are no notifications when I start the game or earn any achievements, nor can I access the pause screen or achievement list, depite having tried both the default and custom hotkeys.

### Reproduction Steps

Logged in to RetroAchievements and played games (Dragon Quest VIII and Katamari Damacy if that matters)

### Expected Behavior

A notification that the game has achievements, notifications when achievements are earned, and the ability to see the list of achievements for the game from a pause screen

### PCSX2 Revision

Nightly - v1.7.3376

### Operating System

macOS (Monteray)

**Comment by stenzek** [comment]
Probably invalid.

Are you sure you have the correct versions of the game? Check the log to see if the server's actually returning valid game info, or look in settings.

**Comment by trvrplk** [comment]
No, I'm 100% sure I have the correct versions of the games because I can earn the achievements. Maybe I'm not explaining it well: the connection to the service works fine, and I can earn achievements, but there's no way to pull up the menu, and I don't get notifications when achievements are earned, or see challenge indicators. So the problem isn't with the game, because the service itself is working: it's just the GUI that shows RetroAchievements-related stuff within the emulator.

**Comment by stenzek** [comment]
Enable a verbose file emulog in the debug menu, and post it here.

**Comment by trvrplk** [comment]
[emuLog.txt](https://github.com/PCSX2/pcsx2/files/9738370/emuLog.txt)

This is just after launching the game

**Comment by refractionpcsx2** [comment]
That's a very old version of pcsx2, it doesn't even have Achievments.

**Comment by trvrplk** [comment]
That's strange that it was using the old version—I had the newer one running. either way, I reinstalled the nightly and have the same problem.

[emulog.txt](https://github.com/PCSX2/pcsx2/files/9739208/emulog.txt)

Also, how do you know I didnt dump all those myself and follow the naming conventions? ;)

**Comment by trvrplk** [comment]
Okay, so here's a log of running it with my personally dumped bios, and photo proof I have the console. Sorry for the complicaiton there, but I hope this can get fixed because it's a legitimate problem in the Mac version.

![IMG_7654](https://user-images.githubusercontent.com/239913/194708669-852fe524-4e65-4014-8056-539e633dec31.jpeg)
![IMG_7655](https://user-images.githubusercontent.com/239913/194708670-9cc8210d-9798-4677-a4f6-aff759e30857.jpeg)
[emulog.txt](https://github.com/PCSX2/pcsx2/files/9739255/emulog.txt)


(Also, I realized the reason the log showed the old version was because it was from a while ago when I first tried the emulator—I hadn't dumped anything at the time, so I just quickly downloaded something to see if it would work. Whoops...)
